server/src/main.rs

```diff
@@ -109,6 +118,43 @@ fn create_session_context() -> SessionContext {
     SessionContext::with_state(state)
 }
 
+// TODO: Move this function to the configuration/metadata component.
+/// If they do not already exist, create the tables used for model table metadata. A
+/// "model_table_metadata" table that can persist model tables is created. A "columns" table that
+/// can save the index of field columns in specific tables is also created. If the tables already
```

Comment:
Maybe rename `model_table_columns` to `model_table_field_columns` since the index of the timestamp and tag columns are not stored in this table.

Reply:
Fixed.